README.md

```diff
@@ -74,6 +77,24 @@ spring.mail.password=$PASSWORD
 All other `spring.mail.*` configurations can be found in the [Spring Documentation E-Mail](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#io.email)
 be viewed.
 
+#### Security Provider
```

Comment:
maybe we can change the wording to be more oidc speak like `OpenID Connect identity provider`